packages/pds/src/services/account/index.ts

```diff
@@ -145,6 +145,17 @@ export class AccountService {
     await sequenceHandleUpdate(this.db, did, handle)
   }
 
+  async updateEmail(did: string, email: string) {
+    const res = await this.db.db
+      .updateTable('user_account')
+      .set({ email })
+      .where('did', '=', did)
+      .executeTakeFirst()
+    if (res.numUpdatedRows < 1) {
+      throw new UserAlreadyExistsError()
```

Comment:
I think this error is probably something more like "user doesn't exist".  Maybe we can even get rid of the `getAccount()` check in the route.

Reply:
ah yup lol. i did a hack copy-paste job